allennlp_models/seq2seq/bart/bart_test.py

```diff
@@ -0,0 +1,82 @@
+from allennlp.data import Vocabulary
+from allennlp.data.token_indexers.pretrained_transformer_indexer import PretrainedTransformerIndexer
+
+# from allennlp.modules.seq2seq_encoders.pytorch_seq2seq_wrapper import GruSeq2SeqEncoder
+from allennlp.nn.util import move_to_device
+
+import torch
+
+# from transformers import BartConfig
+
+from allennlp_models.seq2seq import Bart
+
+
+# TODO: remove this file, it's just for testing.
```

Comment:
Then it should not be in the PR.

We like tests though! How much work is it to add a minimal test for BART?

Reply:
I removed the file, I'll make a test once I am sure everything is working correctly.